tests/BUILD.bazel

```diff
@@ -170,6 +171,7 @@ go_test(
         "windows_test.go",
     ],
     visibility = ["//visibility:public"],
+    x_defs = version_x_defs(),
```

Comment:
@fossedihelm Can you please elaborate on the need for linking the client-go information into the e2e test binary? I couldn't spot any deps related to that change.